gitapp/lib/users_list/users_list.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:gitapp/users_list/user.dart';
+//Lists of the users FGA-3
+class UsersList {
+  List<User> userlist= List.empty(growable: true);
+  List<User> searchedlist= List.empty(growable: true);
+  UsersList(){
+    for(int i=0;i<5;i++)
+    {
+
+      User user = new User(
+          avatarUrl: 'https://i.kym-cdn.com/entries/icons/original/000/035/310/Peepo_Animation_Banner.jpg',
+          username: "User"+i.toString());
+      userlist.add(user);
+    }
+
+  }
+
+void createUsers()  {
+  userlist.clear();
+for(int i=0;i<5;i++)
+  {
+
+    User user = new User(
```

Comment:
Do we have some duplicated functionality ?
the `createUsers` and `UsersList` ?

Reply:
solved